src/app/templates/SettingsGeneral/Components/GeneralSettingLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { T, TID } from 'lib/i18n';
+
+interface Props {
+  titleI18nKey: TID;
+  descriptionI18nKey: TID;
+}
+
+export const GeneralSettingLabel = ({ titleI18nKey, descriptionI18nKey }: Props) => {
+  return (
+    <label className="mb-4 leading-tight flex flex-col" htmlFor="popupEnabled">
```

Comment:
Please either add a property for `htmlFor` value and use it where this component is used or make `htmlFor` equal to `titleI18nKey`